packages/blade/src/components/Checkbox/_decisions/decisions.md

```diff
@@ -0,0 +1,373 @@
+      {children}
+      <CheckboxGroupHelperText type="default | negative" />
+    </CheckboxGroupFieldset>
+  );
+};
+```
+
+## `Checkbox` API
+
+| Prop                | Type       | Required | Default   | Description                                                                                                                                              |
+| ------------------- | ---------- | -------- | --------- | -------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| children            | `string`   | No       | undefined | The text to be rendered as CheckboxLabel                                                                                                                 |
+| helperText          | `string`   | No       | undefined | The helper text to be rendered                                                                                                                           |
+| isChecked           | `boolean`  | No       | false     | state of the checkbox, if explicitly set the checkbox will be controlled                                                                                 |
+| defaultChecked      | `boolean`  | No       | false     | default state of the checkbox, if isChecked is not provided the checkbox will be uncontrolled                                                            |
+| onChange            | `Function` | No       | undefined | The function to be called when the checkbox state changes.                                                                                               |
+| isIndeterminate     | `boolean`  | No       | false     | mixed state of the checkbox                                                                                                                              |
+| name (html native)  | `string`   | No       | undefined | The name of the input field in a checkbox, [useful in form submissions](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#name)            |
+| value (html native) | `string`   | No       | undefined | The value of the input field in a checkbox, [useful in form submissions](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#value) |
+| isDisabled          | `boolean`  | No       | `false`   | Control whether the checkbox is disabled or not.                                                                                                         |
+| isReadOnly          | `boolean`  | No       | `false`   | Control whether the checkbox is readonly or not.                                                                                                         |
+| isInvalid           | `boolean`  | No       | `false`   | Control whether the checkbox is invalid or not.                                                                                                          |
```

Comment:
Since this indicates an error state. Invalid seems too specific whereas error seems a bit generic. It might make a consumer think that setting it as invalid might make the checkbox invalid and make it disappear from the checkbox group. Error is a widely accepted negative state. Thoughts?
```suggestion
| hasError           | `boolean`  | No       | `false`   | Control whether the checkbox is in a error state.                                                                                                          |
```